database/auth.go

```diff
@@ -52,6 +57,146 @@ func (db *Db) saveAccounts(paramsNumber int, accounts []types.Account) error {
 	return nil
 }
 
+// SaveVestingAccounts saves the given vesting account details inside the database, including:
+// type, address, original vesting, endTime, startTime, and vesting periods
```

Comment:
No need to specify what is saved here: 
```suggestion
// SaveVestingAccounts saves the given vesting accounts inside the database
```

Reply:
fixed